### DISCUSSION TITLE
Rancher too heavy for cpx11

### DISCUSSION THREAD

**Discussion by timowevel1** [kube-hetzner/terraform-hcloud-kube-hetzner#322] [Q&A] [answered]
Hey,

for any reason I dont know yet I face a lot of issues trying to get this running, my config is attached.
I enabled Longhorn, enabled Rancher, set the hostname for rancher and thats it, nothing else. Then I create a clean and new project in Hetzner, add the API token and run it. It tells me, that all ressources got added without errors, in Hetzner everything gets created. 

First, when I run kubectl get pod -A, I see the rancher pods restarting each 2-3 minutes, so I cannot access it from outsite. I pointed the domain to the loadbalancer IP. 
cattle-system     rancher-58b9c7b76d-4h9kz                                          0/1     Running            2 (20s ago)     6m40s

Also, the csi controller is restarting all the time and crashing:
kube-system       hcloud-csi-controller-0                                           3/5     CrashLoopBackOff   12 (19s ago)    6m41s

The csi attacher is crashing:
longhorn-system   csi-attacher-59db6bb96c-zlzn7                                     0/1     CrashLoopBackOff   2 (20s ago)     6m40s

And longhorn is restarting:
longhorn-system   longhorn-ui-66cd5fbcf4-b6ltx                                      1/1     Running            2 (6m38s ago)   6m40s

I dont do anything except applying these minor changes to the kube.tf (longhorn, enable rancher). I am running out of ideas, any help is appreciated.
[kube.tf.txt](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/files/9639449/kube.tf.txt)

**Comment by mysticaltech** [comment, marked as answer]
@timowevel1 As per the comments above the rancher section in the kube.tf.example, rancher is quite heavy and needs at least cx21 to run! Otherwise everything will keep on crashing. Hope this helps!

**Reply by timowevel1** [reply]
Thats a good hint, thank you, will try this.
